src/types/lineChart.ts

```diff
@@ -49,7 +47,7 @@ export interface LineChartData {
     /**
      * The color of the stroke.
      */
-    color?: Color;
+    color?: string;
```

Comment:
specify the format in the comment?
```suggestion
    /**
     * The html/hex color of the stroke.
     */
    color?: string;
```